### DISCUSSION TITLE
I want migrate my license from Firefox to Chrome

### DISCUSSION THREAD

**Discussion by BAAL-Beelzebub** [aclap-dev/video-downloadhelper#1558] [License & Purchase requests]
### What's your issue?

Hi,
I would like to migrate my license from Firefox to Chrome.
Please tell me it's possible
I just didn't know that you have an extension for chrome, I thought it's only for Firefox(((((
Can you help me?

### The 10 first digits (only 10) of your license number or email address

b15cad9feb

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Firefox premium so that you can purchase Chrome premium if you want to.
Do you want me to do this ?

**Reply by BAAL-Beelzebub** [reply]
Oh yes, it's definitely better than nothing))
Thanks a lot!

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.
